docs/specification/README.md

```diff
@@ -181,7 +169,7 @@ The Chunk Index records form a coarse index of timestamps to chunk offsets, alon
 
 #### Attachment (op=0x08)
 
-Attachments can be used to attach artifacts such as calibration data, text, or core dumps. Attachment records must not appear within a chunk.
+Attachments can be used to attach artifacts such as calibration data, text, or core dumps. Attachment records must not appear within a chunk, or inside the index data section.
```

Comment:
Why do we call out the index data section for attachment records? We don't do that for Message records.